spec/io/sql_data_source_spec.rb

```diff
@@ -62,5 +66,19 @@
         }.to raise_error(ArgumentError)
       end
     end
+
+    context 'with database file path' do
+      it 'returns a dataframe' do
+        result = Daru::IO::SqlDataSource.make_dataframe(db_file, query)
+        expect(result).to be_a(Daru::DataFrame)
+        expect(result.row[0][:id]).to eq(1)
+        expect(result.row[0][:name]).to eq('Alex')
+      end
+      it 'raises ArgumentError' do
```

Comment:
It is better to have in separate context. In `--document` mode of RSpec, lines 70-82 will generate strangely
```
with database file path
  it returns a dataframe
  it raises ArgumentError
```
...which is not a good description of what's going on.